bootstrap/main.py

```diff
@@ -33,8 +33,10 @@
                             help="Skips the installation of the default compiler")
         parser.add_argument('--noninteractive', action="store_true", default=False,
                             help="Initiates the one click build")
+        parser.add_argument('--run-configuration', action="store_true", default=False,
+                            help="Runs configuration")
         args = parser.parse_args()
-        no_confirm = args.noconfirm or args.noninteractive
+        no_confirm = args.noconfirm or args.noninteractive or args.run_configuration
```

Comment:
We could add `--noconfirm` to the command line instead of hard-coding this. Someone might want to do `--run-configuration` in interactive mode.

Reply:
Updated in https://github.com/apache/nifi-minifi-cpp/pull/1961/commits/9076a8fccca89f2e3ead4fe2bf9e865178fc2077